examples/gallery/line/line-custom-cpt.py

```diff
@@ -0,0 +1,31 @@
+"""
+Line colors with a custom CPT
+-----------------------------
+
+The color of the lines made by :meth:`pygmt.Figure.plot` can be set according to a
+custom CPT and assigned with the *pen* argument.
+
+The custom CPT can be used by setting the plot command's `cmap` argument to *True*. The
+`zvalue` argument sets the z-value (color) to be used from the custom CPT, and the line
+color is set as the z-value by using **+z** when setting the `pen` color.
+
+"""
+
+import numpy as np
+import pygmt
+
+# Create a custom CPT with the batlow CPT and 10 discrete z-values (colors)
+pygmt.makecpt(cmap="batlow", series=[0, 10, 1])
```

Comment:
Please move the `pygmt.makecpt` after calling `pygmt.Figure()`. The current `pygmt.makecpt` creates a CPT for the current session. It works for the current example but may lead to some confusion when users try to adapt the example to more complicated ones. 

See https://docs.generic-mapping-tools.org/dev/makecpt.html#note-on-cpts-in-modern-mode for details.

Reply:
Done